napari/_qt/_qapp_model/_tests/test_file_menu.py

```diff
@@ -207,11 +207,10 @@ def test_open_with_plugin(
 ):
     viewer = make_napari_viewer()
     action, _a = get_open_with_plugin_action(viewer, menu_str)
-    with mock.patch(
-        'napari._qt.qt_viewer.QFileDialog'
-    ) as mock_file, mock.patch(
-        'napari._qt.qt_viewer.QtViewer._qt_open'
-    ) as mock_read:
+    with (
+        mock.patch('napari._qt.qt_viewer.QFileDialog') as mock_file,
+        mock.patch('napari._qt.qt_viewer.QtViewer._qt_open') as mock_read,
+    ):
```

Comment:
❤️

Reply:
I see that you spot one of your favorite python 3.9+ feature?

Reply:
Hahaha indeed. Actually I *was* wondering why these are changed — is it because the file is newly reformatted? Or is the parens around with() actually a new syntax feature??

Reply:
I don't know. I know that I have already read about this in the past, when we talked about ugly formatting, but cannot find now.

Reply:
Actually, it sounds like this was a Python **3.10** feature??? https://github.com/python/cpython/issues/56991 is linked from https://docs.python.org/3/whatsnew/3.10.html. However I notice that it says "officially" allowed, so maybe it was unofficially working in 3.9 and we are now depending on that?

Anyway I think that we should drop 3.9 soon also anyway. (SPEC-0/NEP-29 already support this.)